extras/parse_travis_log.py

```diff
@@ -63,18 +64,20 @@ def process_installcheck(f):
 
 def process_changed_files(f):
     """
-    Run through the file f until a line starts with '+file_names='. After the
-    equal sign is a space separated list of changed files in the PR. Returns that
-    list.
+    Run through the file f until a line starts with '+file_names='.
+    After the equal sign is a space separated list of changed files in the PR.
```

Comment:
The equal sign is followed by a space separated list of changed files in the PR.